python/tvm/driver/tvmc/main.py

```diff
@@ -91,6 +92,11 @@ def _main(argv):
 
     try:
         return args.func(args)
+    except TVMCImportError as err:
+        sys.stderr.write(
+            f'Package "{err.message}" is not installed. ' f'Hint: "pip install tlcpack[tvmc]".'
```

Comment:
I got a final error of `AttributeError: 'TVMCImportError' object has no attribute 'message'` when I ran this, so I think it should just be: 
```suggestion
            f'Package "{err}" is not installed. ' f'Hint: "pip install tlcpack[tvmc]".'
```